lib/wallet/src/Cardano/Wallet/Submissions/TxStatus.hs

```diff
@@ -9,7 +9,7 @@
 Copyright: © 2022 IOHK
 License: Apache-2.0
 
-Data type 'Submissions' for storing a set of submitted transactions.
+Data type 'TxSubmissions for storing a set of submitted transactions.
```

Comment:
```suggestion
Data type 'Submissions' for storing a set of submitted transactions.
```

As the data type is named `Submissions` in the `Cardano.Wallet.Submissions.*` hierarchy, I would favor keeping the same name in the comments. 🤔